File: Assignments/assignment4/Car.java
public class Car {

    private String makeAndModel;
    private int maximumNumberOfPassengers;
    private int numberOfDoors;

    public Car(String makeAndModel, int maximumNumberOfPassengers, int numberOfDoors) {
        this.makeAndModel = makeAndModel;
        this.maximumNumberOfPassengers = maximumNumberOfPassengers;
        this.numberOfDoors = numberOfDoors;
    }

    public void setMakeAndModel(String make) {
        this.makeAndModel = make;
    }

    public String getMakeAndModel() {
        return this.makeAndModel;
    }

    public void setMaximumNumberOfPassengers(int passengers) {
        this.maximumNumberOfPassengers = passengers;
    }

    public int getMaximumNumberOfPassengers() {
        return this.maximumNumberOfPassengers;
    }

    public void setNumberOfDoors(int doors) {
        this.numberOfDoors = doors;
    }

    public int getNumberOfDoors() {
        return this.numberOfDoors;
    }

    @Override
    public String toString() {
        return "Make: " + getMakeAndModel() + "\nCapacity: " + getMaximumNumberOfPassengers() + "\nNumber of Doors: " + getNumberOfDoors();
    }
}

File: Assignments/assignment3/Car.py
class Car:
    def __init__(self, makeAndModel = "none assigned", numberOfDoors = 4, maximumNumberOfPassengers = 5):
        self.makeAndModel = makeAndModel
        self.numberOfDoors = numberOfDoors
        self.maximumNumberOfPassengers = maximumNumberOfPassengers

    def setMakeAndModel(self, string):
        self.makeAndModel = string

    def getMakeAndModel(self):
        return self.makeAndModel
    
    def getMaximumNumberOfPasengers(self):
        return self.maximumNumberOfPassengers
    
    def __str__(self):
        return f"Car: {self.makeAndModel}\nNumber of Doors: {self.numberOfDoors}\nNumber of Passengers: {self.maximumNumberOfPassengers}"